src/black/strings.py

```diff
@@ -236,3 +247,31 @@ def normalize_string_quotes(s: str) -> str:
         return s  # Prefer double quotes
 
     return f"{prefix}{new_quote}{new_body}{new_quote}"
+
+
+def normalize_unicode_escape_sequences(leaf: Leaf) -> None:
+    """Replace hex codes in Unicode escape sequences with lowercase representation."""
+    text = leaf.value
+    prefix = get_string_prefix(text)
+
+    def replace(m: Match[str]) -> str:
+        groups = m.groups()
+        back_slashes = groups[0]
+
+        if len(back_slashes) % 2 == 0 or prefix == "r":
```

Comment:
I'd prefer if you did not instead of comparing to zero :)

Reply:
For what it's worth I prefer `== 0` over relying on implicit boolean behavior.

Reply:
That's fair!